components/Cards/tests/snapshot/MDCCardSnapshotTests.m

```diff
@@ -27,10 +27,9 @@ - (void)testDefaultCard {
 
   // Given
   MDCCard *card = [[MDCCard alloc] initWithFrame:CGRectMake(0, 0, 50, 50)];
-  UIView *backgroundView = [self addBackgroundViewToView:card];
 
-  // Then
-  [self snapshotVerifyView:backgroundView];
+  // When
```

Comment:
This seems like the "Then". You're verifying here.  Maybe it's When + Then?

Reply:
Actually yeah, the `then` is better, `when` can be reserved for applying themes or similar in future tests